NS_0100DA2019044000_Ketsugou_Danshi_Elements_with_Emotions.js

```diff
@@ -0,0 +1,48 @@
+// ==UserScript==
+// @name         [0100DA2019044000] Ketsugou Danshi -Elements with Emotions-
+// @version      1.0.3
+// @author       [Owlie]
+// @description  Yuzu
+// *   	SQUARE ENIX
```

Comment:
Should remove extra spaces here to clean things up a bit
```suggestion
// * SQUARE ENIX
```